src/lolex-src.js

```diff
@@ -710,6 +710,15 @@ function createClock(start, loopLimit) {
 
     if (performancePresent) {
         clock.performance = Object.create(global.performance);
+
+        var proto = Performance.prototype;
+
+        Object
+            .getOwnPropertyNames(global.Performance.prototype)
+            .forEach(function (name) {
+                clock.performance[name] = proto[name];
```

Comment:
@CoderK I just changed this line to fix #200 in my PR #203. Do you see any reason not to replace the methods with stubs? Calling performance.mark() and such started failing after this (PR 160).

Reply:
@fatso83 
I had no the special reason not to replace them with stubs. I just thought that if I have no a specific reason, I should preserve original function as much as possible when mock object is created.

Reply:
Yeah, it seemed like calling those methods failed when not in the original object. At least in some browsers.